source/draw/RasterMonochrome.ooc

```diff
@@ -224,4 +224,10 @@ RasterMonochrome: class extends RasterPacked {
 		}
 		result
 	}
+	writePixelGpu: override func (x, y: Int, color: ColorRgba) {
```

Comment:
I don't see anything in these functions that's unique to a GPU. Isn't it better to leave the `Raster` classes clean and implement the emulation in `CpuMap` or in another class?